taxcalc/utils.py

```diff
@@ -178,11 +177,11 @@ def groupby_weighted_decile(df):
     labels = [range(1,11)]
     # Groupby weighted deciles
     decile_bins = list(range(1, 11))
-    df['wdecs'] = pd.cut(df['cumsum_weights'], bins, labels)
-    return df.groupby('wdecs')
+    df['bins'] = pd.cut(df['cumsum_weights'], bins, labels)
```

Comment:
`groupby_weighted_decile` and `groupby_income_bins` served the purpose of doing a grouping operation. They returned a Pandas GroupBy object, which could be handled generically. These two functions now just add a column named `bins` to the DataFrame, which makes the names confusing. I think we should either name them differently (e.g. `add_weighted_decile_bins`) or do the grouping in the function (I prefer the latter).